local-installation/separate-scripts/start-distributed-analytics.sh

```diff
@@ -1,7 +1,17 @@
 #!/usr/bin/env bash
 
-TAG=2.1.4
+TAG=2.2.0
 REGISTRY=harbor.honeur.org
 
+echo -n "Enter organization name: "
```

Comment:
Should become something like:

read -p "DB schema [omopcdm]: " db_schema
db_schema=${db_schema:-omopcdm}